src/components/hallOfFame/styles.css

```diff
@@ -0,0 +1,25 @@
+.container-hof {
+  padding: 1rem 1rem 0 1rem;
```

Comment:
Could the padding be the same for the rules and hof? You could increase the padding used in the rules or decrease it here.